ballerina/tests/listener_test.bal

```diff
@@ -14,84 +14,82 @@
-        if (accountName.toString() == "WSO2 Inc") {
-            lock {
-                isUpdated = true;
-            }
-            io:println("Updated " + payload.toString());
-        } else {
-            io:println(payload.toString());
-        }
-    }
-
-    remote function onDelete(EventData payload) {
-        string? eventType = payload.metadata?.changeType;
-        if (eventType is string && eventType == "DELETE") {
-            lock {
-                isDeleted = true;
-            }
-            io:println("Deleted " + payload.toString());
-        } else {
-            io:println(payload.toString());
-        }
-    }
-
-    remote function onRestore(EventData payload) {
-        string? eventType = payload.metadata?.changeType;
-        if (eventType is string && eventType == "UNDELETE") {
-            lock {
-                isRestored = true;
-            }
-            io:println("Restored " + payload.toString());
-        } else {
-            io:println(payload.toString());
-        }
-    }
-}
+// service "/data/ChangeEvents" on eventListener {
+//     remote function onCreate(EventData payload) {
+//         string? eventType = payload.metadata?.changeType;
+//         if (eventType is string && eventType == "CREATE") {
+//             lock {
+//                 isCreated = true;
+//             }
+//             io:println("Created " + payload.toString());
+//         } else {
+//             io:println(payload.toString());
+//         }
+//     }
+
+//     remote isolated function onUpdate(EventData payload) {
+//         json accountName = payload.changedData.get("Name");
+//         if (accountName.toString() == "WSO2 Inc") {
+//             lock {
+//                 isUpdated = true;
+//             }
+//             io:println("Updated " + payload.toString());
+//         } else {
+//             io:println(payload.toString());
+//         }
+//     }
+
+//     remote function onDelete(EventData payload) {
+//         string? eventType = payload.metadata?.changeType;
+//         if (eventType is string && eventType == "DELETE") {
+//             lock {
+//                 isDeleted = true;
+//             }
+//             io:println("Deleted " + payload.toString());
+//         } else {
+//             io:println(payload.toString());
+//         }
+//     }
+
+//     remote function onRestore(EventData payload) {
+//         string? eventType = payload.metadata?.changeType;
+//         if (eventType is string && eventType == "UNDELETE") {
+//             lock {
+//                 isRestored = true;
+//             }
+//             io:println("Restored " + payload.toString());
+//         } else {
+//             io:println(payload.toString());
+//         }
+//     }
+// }
 
 // Using direct-token config for client configuration
 Client lisbaseClient = check new (sfConfigRefreshCodeFlow);
 string testRecordId = "";
 
 @test:Config {
-    enable: true
+    enable: false
```

Comment:
Any particular reason to disable the listeners tests on U10?

Reply:
Yeah Maheshika has not set the secrets correctly so the authentication fails

Reply:
Ack. In that case, better to create and issue for tracking purposes ftm. Please add the issue link in here once created.

Reply:
https://github.com/ballerina-platform/ballerina-library/issues/6908
https://github.com/ballerina-platform/ballerina-library/issues/6909